doc/api/process.md

```diff
@@ -2260,6 +2260,29 @@ process.kill(process.pid, 'SIGHUP');
 When `SIGUSR1` is received by a Node.js process, Node.js will start the
 debugger. See [Signal Events][].
 
+## `process.loadEnvFile(path)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active development
+
+* `path` {string | undefined}
```

Comment:
IMO this should accept the same input as other built-in accepting paths (e.g. `node:fs` methods)
```suggestion
* `path` {string | URL | Buffer | undefined}
```